### PR TITLE
update Go version for CI

### PR DESCRIPTION
**Pull request by fabxc**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ sudo: false
 language: go
 
 go:
- - 1.5.4
- - 1.6.2
+ - 1.6
```
Comment (beorn7) [inline .travis.yml]: Does Travis use the latest 1.6 in that case, or just the original 1.6.0?

Comment (sdurrheimer) [inline .travis.yml]: If I'm not mistaken `1.6` means `1.6.0` (https://github.com/travis-ci/gimme/blob/master/gimme#L95)

Comment (fabxc) [inline .travis.yml]: The former is what this scheme of providing a version generally implies.
Couldn't find anything explicit in the Travis docs via googling though.

Comment (beorn7) [inline .travis.yml]: It's more a problem with how Go treats versioning.
They call 1.6.0 in fact just 1.6, so I guess we have to explicitly specify the latest bugfix version.

Comment (fabxc) [inline .travis.yml]: Ah, bummer. That doesn't seem like a reasonable pattern to me =/

**Comment by beorn7** [comment]
:+1:

**Pull request merged**